### PR TITLE
Add Sphinx documentation

### PR DESCRIPTION
**Pull request by SamWhited**
Adds some sphinx documentation (and a makefile rule to generate it). To generate the docs run:

    make docs

This will also be run by Travis.

### CODE REVIEW

File: requirements-test.txt [hunk 1]
```diff
@@ -2,3 +2,5 @@ coverage
 flake8
 pytest
 tox
+sphinx
```
Comment (campaul) [inline requirements-test.txt]: Why is sphinx needed for tests?

Comment (SamWhited) [inline requirements-test.txt]: On May 18, 2015 7:37:32 PM CDT, Cameron Paul <notifications@github.com> wrote:
>> @@ -2,3 +2,5 @@ coverage
>>  flake8
>>  pytest
>>  tox
>> +sphinx
>
>Why is sphinx needed for tests?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/photoshell/rawkit/pull/13/files#r30560405

Travis builds the docs now to make sure we don't break them.
-- 
Sam Whited
pub 4096R/54083AE104EA7AD3

**Comment by campaul** [comment]
`make docs` fails if you don't manually install sphinx right now.

**Comment by SamWhited** [comment]
Odd, it should be using the one out of your venv. I'll take a look at it.

**Comment by SamWhited** [comment]
Seems to work for me (uninstalled sphinx, git clean -dxf-ed, and ran make docs).

**Comment by SamWhited** [comment]
Squashed and merged.

**Pull request closed without merging**